packages/react/src/index.ts

```diff
@@ -265,10 +265,13 @@ export abstract class Block<P = {}, S = {}> extends Component<EntityProps<P>, S>
     }
 
     public render(): ReactNode {
-        const { props, state } = this;
-        const optionalyReplaced = this.replace(props, state);
+        const tag = this.tag(this.props, this.state);
+        const children = this.content(this.props, this.state);
+        const attrs = this.attrs(this.props, this.state);
+        const style = this.style(this.props, this.state);
+        const extendedAttributes = { ...attrs, style: { ...attrs.style, ...style } };
 
-        return this.wrap(props, state, optionalyReplaced);
+        return this.prerender(tag, extendedAttributes, children);
```

Comment:
It's useful method without `wrap` and `replace`. Let's Remove it too ;)

Reply:
Nope, because prerender get context from `Block` and `Elem`

Reply:
I know my friend) Let's move logic here ;)